config/custom.js

```diff
@@ -19,5 +19,5 @@ module.exports.custom = {
   // mailgunSecret: 'key-testkeyb183848139913858e8abd9a3',
   // stripeSecret: 'sk_test_Zzd814nldl91104qor5911gjald',
   // …
-
+  baseURL: process.env.BASE_URL || 'http://localhost:1337',
```

Comment:
Question: Is there a reason we wouldn't use the dynamically generated URL? (such as is generated in the invite controller?)

Reply:
Ah! Never mind. Saw the note.